packages/shared/components/popups/RemoveProposalPopup.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="typescript">
+    import { Button, Text, TextType, TextHint } from 'shared/components'
+    import { ButtonVariant } from 'shared/components/enums'
+    import { handleError } from '@core/error/handlers/handleError'
+    import { selectedProposal } from '@core/governance'
+    import { localize } from '@core/i18n'
+    import { deregisterParticipationEvent } from '@core/profile-manager/api'
+    import { GovernanceRoute, governanceRouter } from '@core/router'
+    import { closePopup } from '@auxiliary/popup'
+    import { showAppNotification } from '@auxiliary/notification'
+
+    function handleCancel(): void {
+        closePopup()
+    }
+
+    async function handleConfirm(): Promise<void> {
+        try {
+            await deregisterParticipationEvent($selectedProposal.id)
+            showAppNotification({
+                type: 'success',
+                message: localize('views.governance.proposals.successRemove'),
+                alert: true,
+            })
+            closePopup()
+            $governanceRouter.goTo(GovernanceRoute.Proposals)
```

Comment:
Minor thing: could you either use `$governanceRouter.previous()` or `$governanceRouter.reset()`? Otherwise this propsal's route would be still on the previous routes' stack.